### DISCUSSION TITLE
Working on Raspberry Pi? --- funktioniert das auch mit einem Raspberry Pi

### DISCUSSION THREAD

**Discussion by Maexle-38** [jomjol/AI-on-the-edge-device#2726] [Q&A]
Hello community,

is it possible to use AI-on-the-edge-device on a Raspberry Pi?

Hallo zusammen,
ist das AI-on-the-edge-device auch auf einem Raspberry Pi lauffähig? Könnt ihr mir damit weiterhelfen?

**Comment by locutus13** [comment]
Wondering the same just in this moment. I currently have some PI's with the default camera module that record an image every hour into a folder. All I would need to do is to load that image and do the "AI" part ... couldn't find a tutorial via Google.
Now I wonder if it is worth - just trying / adapting the code - but my coding skills end at PHP and HTML.

**Reply by locutus13** [reply]
and then I found this: https://github.com/jomjol/water-meter-system-complete

**Reply by Maexle-38** [reply]
Hello,

the RPi is about 4 meters away from the watermeter --> so i thought it could be possible to use an USB-Webcam on the RPi.
this is the watermeter: https://www.kamstrup.com/de-de/wasserzaehlerloesungen/intelligente-wasserzaehler/meters/multical-21

the Pi is already connected to a "smarthome" via KNX

**Reply by locutus13** [reply]
There is this code:
https://github.com/jomjol/neural-network-digital-counter-readout

but it is old enough that recent changes in python and raspbian/debian prevent it form working without setting up a virtual environment... I think ... bit too complex for me at the moment.

**Comment by jomjol** [comment]
This code is not maintained any more and also not planned to be reactivated.
